controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/configserver/ConfigServer.java

```diff
@@ -53,7 +54,9 @@
      */
     InputStream getLogs(DeploymentId deployment, Map<String, String> queryParameters);
 
-    List<ClusterMetrics> getMetrics(DeploymentId deployment);
+    List<ClusterMetrics> getDeploymentMetricsV1(DeploymentId deployment);
+
+    JSONObject getProtonMetricsV1(DeploymentId deployment);
```

Comment:
Instead of JSONObject, return a ProtonMetrics object, which can do validation and aggregation

Reply:
Implemented